### PR TITLE
RK 273 Build Check Answer Page

### PR DESCRIPTION
**Pull request by shiromi**

### CODE REVIEW

File: conf/app.routes [hunk 1]
```diff
@@ -169,11 +169,12 @@ POST       /your-contact-details                         controllers.YourContact
 GET        /change-your-contact-details                  controllers.YourContactDetailsController.onPageLoad(mode: Mode = CheckMode)
 POST       /change-your-contact-details                  controllers.YourContactDetailsController.onSubmit(mode: Mode = CheckMode)
 
+GET        /check-answers-Individual1                          controllers.IndividualCheckYourAnswersController.onPageLoad
```
Comment (wolfendale) [inline conf/app.routes:172]: typo here

File: app/viewmodels/checkAnswers/IndividualContactDetailsSummary.scala [hunk 2]
```diff
@@ -31,9 +31,11 @@ object IndividualContactDetailsSummary {
   def row(index: Int, answers: UserAnswers)(implicit messages: Messages): Option[SummaryListRow] =
     answers.get(IndividualContactDetailsPage(Index(index))).map {
       answer =>
-        val value = List(answer.landlineNumber, answer.mobileNumber, answer.email).flatten.map(
-          HtmlFormat.escape
-        ).mkString("<br />")
+        val value =
+          List("Landline" -> answer.landlineNumber, "Mobile" -> answer.mobileNumber, "Email" -> answer.email) flatMap {
```
Comment (wolfendale) [inline app/viewmodels/checkAnswers/IndividualContactDetailsSummary.scala:35]: These labels will need to use `messages`

File: app/viewmodels/checkAnswers/IndividualNameSummary.scala [hunk 3]
```diff
@@ -31,9 +31,15 @@ object IndividualNameSummary {
   def row(answers: UserAnswers, index: Int)(implicit messages: Messages): Option[SummaryListRow] =
     answers.get(IndividualNamePage(Index(index))).map {
       answer =>
-        val value = List(answer.firstName, answer.middleName, answer.lastName, answer.aliases).flatten.map(
-          HtmlFormat.escape
-        ).mkString("<br/>")
+        val value = List(
+          "First Name"  -> answer.firstName,
```
Comment (wolfendale) [inline app/viewmodels/checkAnswers/IndividualNameSummary.scala:35]: These labels will need to use `messages`

File: app/viewmodels/checkAnswers/ReferenceNumbersSummary.scala [hunk 4]
```diff
@@ -37,7 +37,7 @@ object ReferenceNumbersSummary {
 
         SummaryListRowViewModel(
           key = "referenceNumbers.checkYourAnswersLabel",
-          value = ValueViewModel(HtmlContent(value)),
+          value = ValueViewModel(HtmlContent("VAT Registration number: <br /> " + value)),
```
Comment (wolfendale) [inline app/viewmodels/checkAnswers/ReferenceNumbersSummary.scala:40]: This label will need to use `messages`

File: app/viewmodels/checkAnswers/YourContactDetailsSummary.scala [hunk 5]
```diff
@@ -16,31 +16,44 @@
...
-  def row(answers: UserAnswers)(implicit messages: Messages): Option[SummaryListRow] =
-    answers.get(YourContactDetailsPage).map {
+  def rows(answers: UserAnswers)(implicit messages: Messages): List[SummaryListRow] =
+    answers.get(YourContactDetailsPage).toList.flatMap {
       answer =>
-        val value = HtmlFormat.escape(answer.FirstName).toString + "<br/>" + HtmlFormat.escape(answer.LastName).toString
+        def msg(key: String) = messages("yourContactDetails." + key)
+
+        val values = List(
+          "firstName"     -> Some(answer.FirstName),
```
Comment (wolfendale) [inline app/viewmodels/checkAnswers/YourContactDetailsSummary.scala:36]: These labels will need to use `messages`

**Review by wolfendale — CHANGES_REQUESTED** [review]

File: app/viewmodels/checkAnswers/IndividualNameSummary.scala (hunk 3 above)
Comment (shiromi) [reply app/viewmodels/checkAnswers/IndividualNameSummary.scala:35]: This is coming from messgae

**Review by wolfendale — APPROVED** [review]

**Pull request merged**